### ISSUE TITLE
[generate:entity:content] hook_theme not generated for entities without bundles

### ISSUE THREAD

**Issue by HaleFx** [hechoendrupal/DrupalConsole#1767]
Looks like it gets generated for content entities with bundles, but not for entities without bundles. The entity won't run through the twig template without it.

**Comment by cosmicdreams** [comment]
@HaleFx Yes, this feature was originally made for traditional content types.  Are you expecting it to work for custom entities' bundles?

At one point it was considered calling this feature generate:entity:bundle.  

What do you think this feature should be doing that it isn't?

**Comment by HaleFx** [comment]
If I run generate:entity:content and say "no" to the bundle option, the ENTITY_TYPE.page.inc and ENTITY_TYPE.html.twig files are both created, but hook_theme is not created in the module file (actually, no MODULE.module file is created at all), so neither one is used. If you say "yes" to the bundle option, hook_theme is created (although it looks like right now the MODULE.module file is missing an opening `<?php`), so both of the above files are used. I think that hook_theme (and the .module file) should be added in both situations since the files are created and can be useful either way.

Does that make sense?

**Comment by cosmicdreams** [comment]
Yes,  I don't recall coding it that way but know the place to look.  ETA: Wednesday.

**Comment by frega** [comment]
@cosmicdreams @HaleFx that sounds like a regression or possibly a missing feature, not sure which. I certainly see the point to add an appropriate theme function. 

IIRC there's a small issue relating to hook_theme-entries because DrupalConsole cannot "modify" existent hook_theme-implementations so you end up w/ a new hook_theme-implementations.
